R/tool-environment.R

```diff
@@ -61,8 +61,16 @@ btw_tool_describe_environment <- function(
   item_desc <- map(env_item_names, function(item_name) {
     item <- env_get(environment, item_name)
 
+    if (identical(class(item), "character")) {
+      # Only string literals passed through btw() hit `btw_this.character().
+      # We rely on `dots_list()` turning `"foo"` into `list('"foo"' = "foo")`.
+      if (!identical(item_name, sprintf('"%s"', item))) {
+        item <- btw_returns_character(item)
+      }
+    }
```

Comment:
This part might be a little brittle, and I'd be happy to find another way to identify a string literal. That said, you'd have to do something like this to trick it:

```r
`"beep"` <- "beep"
btw(`"beep"`)
```